### PR TITLE
MDidtouri spec has correct data types

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: spec/integration/mixins/id_to_uri_spec.rb [hunk 1]
```diff
@@ -24,12 +24,14 @@ class SomeTestClass
 
   it 'should throw BadRequest for -1' do
     expect do
-      SomeTestClass.identifier_to_uri(@opts, -1)
-    end.to raise_error(RestClient::BadRequest)
+      SomeTestClass.identifier_to_uri(@opts, '-1')
+    end.to raise_error(RestClient::BadRequest) { |error|
+      error.response.should match /does not match with value '-1'/
```
Comment (yenkins) [inline spec/integration/mixins/id_to_uri_spec.rb]: Lint/AmbiguousRegexpLiteral: Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/541/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/299/ : FAILURE in 29s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/606/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/542/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/300/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/607/ : SUCCESS in 30s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/197/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/301/ : SUCCESS in 27s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/608/ : SUCCESS in 30s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1098/ : SUCCESS in 43m 29s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/855/ : SUCCESS in 22m 29s

**Pull request merged**